src/relay.cpp

```diff
@@ -1089,8 +1075,7 @@ void loop_relay(std::unordered_map<std::string, relay_config> &vlans) {
 
         update_vlan_mapping(vlan.first, config_db);
 
-        std::string counterVlan = counter_table;
-        initialize_counter(vlan.second.state_db, counterVlan.append(vlan.second.interface));
+        initialize_counter(vlan.second.state_db, vlan.second.interface);
```

Comment:
Changes look good, i also suggest clearing out the entire COUNTERS Table, during init. That'll fix this issue as well `https://github.com/sonic-net/sonic-buildimage/issues/15047`

Reply:
> https://github.com/sonic-net/sonic-buildimage/issues/15047

Looks like the issue is handled by https://github.com/sonic-net/sonic-utilities/pull/2852, not required to clear DHCPv6_COUNTERS table here. #Resolved